### PR TITLE
Get full path to source in publish

### PR DESCRIPTION
**Pull request by danghai**
I make PR for getting full path to source in publish

### CODE REVIEW

**Comment by danghai** [comment]
@veruu . In `cmd_publish`, it is going to publish the `buildinfo` and `buildconf`. Getting full path `buildinfo` and `buildconf` in configuration `cfg` can fix this issue? I am not sure about it.

File: skt/publisher.py [hunk 1]
```diff
@@ -58,20 +58,16 @@ class CpPublisher(Publisher):
...
 class ScpPublisher(Publisher):
     TYPE = 'scp'
 
     def publish(self, source):
-        # FIXME Move expansion up the call stack, as this limits the class
-        # usefulness, because tilde is a valid path character.
         subprocess.check_call(["scp",
-                               os.path.expanduser(source),
+                               source,
                                self.destination])
```
Comment (veruu) [inline skt/publisher.py]: Since the full call fits into one line now, can you merge those lines so it doesn't take up so much space?

Comment (danghai) [reply skt/publisher.py]: Yeah, I have intend to ask you about make absolute path for `tarpkg`, but it seems to me the `load_config` become longer. 
Ah I got what you mean. We should get absolute path for all configuration early. And then, we are not worry about path for later design

**Comment by danghai** [comment]
Hmm not sure why it has the change from previous patch

**Comment by spbnick** [comment]
Thank you, Hai, Veronika! This looks good, merging. However, I would've been happier if the commit subject matched the change better :)

**Pull request merged**